src/assets/javascripts/sharing-button.mjs

```diff
@@ -7,16 +7,20 @@ export default function sharingButton () {
   var urlString
   var onClick
 
+  // return early if no of the required APIs are available
+  if (!navigator.share && !document.queryCommandSupported('copy')) return
```

Comment:
Minor: these looks like they belong in the previous commit.

Reply:
Agreed and addressed in https://github.com/alphagov/notifications-govuk-alerts/pull/105/commits/f1f3493e607847e8d7b738af605c811eacbe1415